event-streams/src/event-streams-v1.0-wd01.md

```diff
@@ -0,0 +1,547 @@
+Considerations", BCP 72, RFC 3552, DOI 10.17487/RFC3552, July 2003,
+https://www.rfc-editor.org/info/rfc3552.
+
+# 2 Concepts and default behaviors
+
+The following terminology is used in this specification to clarify roles. These
+definitions are narrower than their use in the core AMQP specification.
+
+- `producer` is the sender of events into the event log.
+- `consumer` is the receiver of events from the event log.
+- `event log node` is where AMQP links of senders and receivers attach.
+
+
+## 2.1 Partition support
+
+An event log managed by the event stream node MAY be split into multiple
+parallel partitions that are reachable under the same network address. A single
+event log might also be shared by multiple AMQP containers/nodes that each have
+a separate network address and are each responsible for one or more partitions.
+
+The association between a producer/consumer and a partition MAY be chosen using
+link properties when the link is being attached. The bi-directional negotiation
+capabilities of AMQP apply here: A consumer MAY request being associated with a
+particular partition, but the responding event log node MAY also assign a
+partition if none has been requested.
+
+A producer's `send` link being attached to an event log node MAY request
+association with a specific partition using the [`event-streams-partition` link
+property](#211-event-streams-partition-link-property). If the requested
+partition does not exist, attaching the link MUST be rejected by the event log
+node. The event log node MAY assign a partition when none has been requested. It
+MUST NOT assign a partition other than the one that has been requested.
+
+If no partition is assigned, the default behavior MUST be for a `send` link to
+be partition agnostic and for any partition-related hints to flow via delivery
+annotations, which are defined further below in this section.
+
+A consumer's `receive` link being attached to an event log node MAY request
+association with a specific partition using the the [`event-streams-partition`
+link property](#211-event-streams-partition-link-property). If the requested
+partition does not exist, attaching the link MUST fail. If the event log is
+partitioned and no association has been requested, the event log node MUST
+associate a partition with the link.
+
+For both `send` and `receive` links, the effective associated partition MUST be
+expressed with the [`event-streams-partition` link
+property](#211-event-streams-partition-link-property) set by the event log node.
+
+### 2.1.1 event-streams-partition link property
+
+The `event-streams-partition` link property is a `symbol` value and describes
+the partition with which the link is associated. When set by the producer or
+consumer, the property value is a request for association with the given
+partition. When set by the event log node, the value is the effective
+association.
+
+When not set for a producer 'send' link, the link is partition-agnostic.
```

Comment:
I am thinking more about how Apache Kafka works here for example.
The logic to determine the destination partition is on the producer which reflected here means that a producer should always attach a link specifying a partition.
If we allow to not set the partition, it means that we are offloading the logic of the destination partition to the event log node, right? It could make sense if there is an intermediary (like a gateway/bridge) in the middle, having the logic for this.